homework/homework11/bank_deposit.py

```diff
@@ -32,8 +38,29 @@ class Customer:
     """Initialize customer's"""
     def __init__(self, name: str):
         """This method outputs the name of the bank's customer"""
+        if not name:
+            raise ValueError("Name cannot be empty")
+        if not all(char.isalnum() or char.isspace() for char in name):
+            raise ValueError("Name can only contain alphanumeric characters"
+                             " and spaces")
+        if len(name) > 255:
+            raise ValueError("Name is too long")
         self.name = name
 
+    def __str__(self):
+        return self.name
+
+
+if __name__ == "__main__":
+    person_1 = Customer('Lionel Messi')
+    person_2 = Customer('LeBron James')
+
+    logging.info(f'Customer created: {person_1.name}')
+    deposit_person1 = Bank(1000000, 120, 10)
+    logging.info(deposit_person1)
+    logging.info(f'Monthly capitalization: '
+                 f'{deposit_person1.calculate_monthly_capitalization()}')
+
 
 if __name__ == "__main__":
     person_1 = Customer('Lionel Messi')
```

Comment:
second `if __name__ == "__main__":` condition? Remove lines number 65-66

Reply:
deleted the extra user call, my mistake, I was making edits at night and didn't notice it. Sorry